### PR TITLE
Use version in external URL

### PR DESCRIPTION
**Pull request by vilyaair**
Some external URLs in the document directly refer to the master branch, and they should be pointed to the corresponding tag.

I utilize sphinx.ext.extlinks to achieve this goal. However, I wonder if it is appropriate to use names 'blob' and 'tree', and actually github automatically redirects to correct URL whichever original URL is blob or tree.

### CODE REVIEW

**Comment by niboshi** [comment]
jenkins, test this please

File: docs/source/tutorial/function.rst [hunk 1]
```diff
@@ -608,4 +608,4 @@ offers a utility function :func:`chainer.gradient_check.check_backward` that fol
    unittest.TextTestRunner().run(suite)
 
 
-You can find many examples of function tests under ``tests/chainer_tests/function_tests`` directory.
+You can find many examples of function tests under :tree:`tests/chainer_tests/function_tests` directory.
```
Comment (niboshi) [inline docs/source/tutorial/function.rst]: `function_tests` -> `functions_tests`

Comment (vilyaair) [reply docs/source/tutorial/function.rst]: I appreciate it. I should scrutinize my code more carefully.

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by niboshi** [comment]
LGTM!
Thank you very much!

**Pull request merged**